tests/unit/test_interval_regex.py

```diff
@@ -0,0 +1,34 @@
+import re
+import pytest
+
+cron_regex = re.compile(
+    r"^(?:-1|\d+(?:\.\d+)?|(((\d+,)+\d+|(\d+[/-]\d+)|\d+|\*(\/\d*)?) ?){5})$"
```

Comment:
Can we import it from schema.json or somewhere else so that if we ever decide to change the regex, we just have to change at one place?

Reply:
Declared as global var, not a perfect solution, as still in schema we need it to be defined additionally, but I don't want to parse .json to create sourcecode